src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -685,6 +687,29 @@ bool layout_optimizer::convolution_b_fs_yx_fsv16_opt(const layout& input_layout,
     return false;
 }
 
+static bool has_reorder_before_mvn(const program_node& node, size_t cur_depth, size_t max_depth, uint64_t reorder_size_threshold = 0) {
+    // MVN with rank size 3 always requires Reorder and Reshape. Due to this pattern, too many Reorder may occur when used with Convolution,
+    // which may performance degradation. It stand out in Stable-Diffusion Unet and Decoder.
```

Comment:
nit: `which may performance degradation` -> `which may cause performance degradation`
`It stand out` -> `It stands out`

Reply:
Applied.